### ISSUE TITLE
ReportStream ORM/OML Transformations Losing Data

### ISSUE THREAD

**Issue by JohnNKing** [CDCgov/trusted-intermediary#900]
# Bug

## Describe the Bug

ReportStream's ORM to FHIR and FHIR to OML transformations are losing data from a variety of fields, including:

- PID-3.4
- PID-8.2
- PID-8.3
- PID-10
- PID-11.7
- PID-18
- PID-22
- NK1-2
- NK1-3.3
- NK1-5
- NK1-33
- ORC-2.2
- ORC-12
- OBR-2
- OBR-4
- OBR-7
- OBR-16
- OBX-3 (all except CE/CWE datatype)
- OBX(2)-6 and OBX(4)-6 (NM datatypes (units))
- OBX(8)-2 (XAD datatype)
- OBX(8)-5 (XAD datatype)

I suspect this was introduced by: https://github.com/CDCgov/prime-reportstream/pull/12718

For more details, see https://slack.com/archives/C048S3GPSQG/p1707861774688789

Also ensure an error isn't caused when the order has no OBX segments. (i.e. make OBX not required for ORM messages). Potentially relevant: https://github.com/CDCgov/prime-reportstream/blob/0eb5b42c5a8ccb2d7876fbee7441c790eaf5b828/prime-router/src/main/resources/metadata/hl7_mapping/OML_O21/OML_O21-base.yml#L62

Recently transformed ORM (from 9/11/2024): https://www.slack.com/archives/C044P2CBEKS/p1726088886147019?thread_ts=1726088335.991899&cid=C044P2CBEKS

## Impact

ORM orders are lacking key pieces of information for NBS.

## To Reproduce

1. Generate a JWT using the Flexion staging API key for ReportStream
2. Use the ReportStream login endpoint to get a session JWT
3. Send the following to the Water's endpoint: https://github.com/CDCgov/trusted-intermediary/blob/main/examples/MN/003_MN_ORM_O01_NBS.hl7
4. 5 min later, use the RS admin UI's daily-data page to download the resulting HL7 file

## Expected Behavior

Expected HL7 output should be the following (or equivalent):

```
MSH|^~\&|Epic^1.2.840.114350.1.13.145.2.7.2.695071^ISO|Centracare^centracare.com^DNS|NATUS^natus.health.state.mn.us^DNS|MN Public Health Lab^2.16.840.1.114222.4.1.10080^ISO|20230506052916-0500||OML^O21^OML_O21|31808297|P|2.5.1|||AL|AL|||||LAB_PRU_COMPONENT^^2.16.840.1.113883.9.82^ISO~LAB_TO_COMPONENT^^2.16.840.1.113883.9.22^ISO
PID|1||11102779^^^^MR||SMITH^BB SARAH^^^^^L|SMITH^SADIE^S|20230504131023-0500|M|||555 STATE HIGHWAY 13^^DEER CREEK^IG^56527-9657^USA^H^^OTTER TAIL||^^PH^^^763^555555^^(763)555-5555|||||||||||N||||||N
NK1|1|SMITH|MTH^mother|555 STATE HIGHWAY 13^^DEER CREEK^IG^56527-9657^USA^H^^OTTER TAIL|^PRN^PH
ORC|NW|421832901^EPIC|||unknown||||20230506102913+0000
OBR|1|||54089-8^Newborn screening panel American Health Information Community (AHIC)^LN|||20230506050000+0000||||P
OBX|1|NM|8339-4^BIRTH WEIGHT MEASURED^LN|1|1769.8593|g^gram^UCUM|||||unknown|||20230506050000-0500
OBX|2|NM|57714-8^OBSTETRIC ESTIMATION OF GESTATIONAL AGE^LN|1|32|wk^week^UCUM|||||unknown|||20230506050000-0500
OBX|3|CWE|57713-0^INFANT FACTORS THAT AFFECT NEWBORN SCREENING INTERPRETATION^LN|1|LA12419-0^INFANT IN NICU AT TIME OF SPECIMEN COLLECTION^LN||||||unknown|||20230506050000-0500
OBX|4|CWE|MNDSIB^DOES THE BABY HAVE A DECEASED SIBLING?^99MDH|1|N^No^99MDH||||||unknown|||20230506050000-0500
OBX|5|CWE|MNDEFECT^DOES THE BABY HAVE BIRTH DEFECTS?^99MDH|1|N^No^99MDH||||||unknown|||20230506050000-0500
OBX|6|CWE|MNFAM^FAMILY HISTORY OF DISORDER ON MN SCREENING PANEL^99MDH|1|N^No^99MDH||||||unknown|||20230506050000-0500
OBX|7|CWE|67704-7^FEEDING TYPES^LN|1|LA12418-2^TPN^LN||||||unknown|||20230506050000-0500
OBX|8|CWE|67704-7^FEEDING TYPES^LN|2|LA16914-6^BREAST MILK^LN||||||unknown|||20230506050000-0500
OBX|9|CWE|57723-9^UNIQUE BAR CODE NUMBER OF CURRENT SAMPLE^LN|1|0516199364||||||unknown|||20230506050000-0500
OBX|10|ST|62328-0^DISCHARGE PROVIDER PRACTICE PHONE NUMBER^LN|1|Daniel Davis/218-555-1000||||||unknown|||20230506050000-0500
OBX|11|ST|62324-9^POST-DISCHARGE PROVIDER NAME^LN|1|DAVIS, DANIEL, MD||||||unknown|||20230506050000-0500
OBX|12|ST|62326-4^POST-DISCHARGE PROVIDER PRACTICE NAME^LN|1|Lakeridge Health Revere||||||unknown|||20230506050000-0500
SPM|1|||440500007^Blood spot specimen^SCT|||||||||||||20230506050000
```

## Tasks
- [x] Reach out to ReportStream to better understand the current status
- [x] Test some of our sample ORMs against the ReportStream PR: https://github.com/CDCgov/prime-reportstream/pull/15910
- [x] Review tests that are part of this PR; determine if those tests covers all of our data needs (or not)
- [x] Provide feedback to RS team if we find any gaps
~- [ ] Look into possibility of adding assertions in the RS Integration Test framework to catch mapping issues~
- [x] Remove OBR-11 transformation from etor-oml-receiver-transform.yml

## Screenshots

n/a

## Logs

n/a

## Version

Current master branch

## Additional Context

n/a

**Comment by eaugustini** [comment]
While reviewing AL test order scenarios output, I found that these fields were missing in the output OML
PID-3.4
PID-8.2
PID-8.3
PID-10
PID-11.7
PID-18
PID-22
NK1-2
NK1-3.3
NK1-5
NK1-33
ORC-2.2
ORC-12
OBR-2
OBR-4
OBR-7
OBR-16
OBX-3 (all except CE/CWE datatype)
OBX(2)-6 and OBX(4)-6 (NM datatypes (units))
OBX(8)-2 (XAD datatype)
OBX(8)-5 (XAD datatype)

**Comment by JohnNKing** [comment]
Original list of missing fields: - OBX-3
- OBX-6
- OBX-5-3
- OBR-4
- OBR-7
- ORC-21
- SPM-4
- SPM-17

**Comment by JohnNKing** [comment]
Existing ReportStream epic:
https://app.zenhub.com/workspaces/platform-6182b02547c1130010f459db/issues/gh/cdcgov/prime-reportstream/14958

PR: https://github.com/CDCgov/prime-reportstream/pull/1591

**Comment by basiliskus** [comment]
PR in RS: https://github.com/CDCgov/prime-reportstream/pull/15910

**Comment by jbiskie** [comment]
This is the E2E output of the sample message linked in the summary when run against the `OML-Updates-Base` feature branch post-PR #15910 merge.

```
MSH|^~\&|Epic^1.2.840.114350.1.13.145.2.7.2.695071^ISO|Centracare^centracare.com^DNS|NATUS^natus.health.state.mn.us^DNS|MN Public Health Lab^2.16.840.1.114222.4.1.10080^ISO|20230506052916-0500||OML^O21^OML_O21|31808297|P|2.5.1|||AL|AL|||||LAB_PRU_COMPONENT^^2.16.840.1.113883.9.82^ISO~LAB_TO_COMPONENT^^2.16.840.1.113883.9.22^ISO
PID|1||11102779^^^CRPMRN^MR||SMITH^BB SARAH^^^^^L|SMITH^SADIE^S|20230504131023-0500|M|||555 STATE HIGHWAY 13^^DEER CREEK^MN^56527-9657^USA^H^^OTTER TAIL||^^PH^^^763^5555555^^(763)555-5555|||||||||||N|1|||||N
ORC|NW|421832901^EPIC^1.2.840.114350.1.13.145.2.7.2.695071^ISO|||||||20230506052913-0500|||1265136360^JONES^JANE^^^^^^NPI^L^^^NPI|||||||||ST. CLOUD HOSPITAL^^^^^CMS^NPI^^^1043269798~ST. CLOUD HOSPITAL^^^^^MN Public Health Lab^Submitter ID^^^739
OBR|1|421832901^EPIC^1.2.840.114350.1.13.145.2.7.2.695071^ISO|||||||||P
```

**Comment by jbiskie** [comment]
ReportStream HL7 to FHIR output of https://github.com/CDCgov/trusted-intermediary/tree/main/examples/MN/003_MN_ORM_O01_NBS.hl7 when run on the OML-Updates-Base feature branch:

```
{
	"resourceType": "Bundle",
	"id": "1729025485464275004.9491bc69-2f81-4abf-b7d7-386491246e4f",
	"meta": {
		"lastUpdated": "2024-10-15T20:51:25.464+00:00"
	},
	"identifier": {
		"system": "https://reportstream.cdc.gov/prime-router",
		"value": "31808297"
	},
	"type": "message",
	"timestamp": "2024-07-02T05:29:16.000+00:00",
	"entry": [
		{
			"fullUrl": "MessageHeader/1729025485468024474.58339793-4739-46be-ad3d-dc85eedd53df",
			"resource": {
				"resourceType": "MessageHeader",
				"id": "1729025485468024474.58339793-4739-46be-ad3d-dc85eedd53df",
				"meta": {
					"tag": [
						{
							"system": "http://terminology.hl7.org/CodeSystem/v2-0103",
							"code": "T"
						}
					]
				},
				"extension": [
					{
						"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/msh-message-header",
						"extension": [
							{
								"url": "MSH.7",
								"valueString": "20240702052916"
							},
							{
								"url": "MSH.15",
								"valueString": "AL"
							},
							{
								"url": "MSH.16",
								"valueString": "AL"
							},
							{
								"url": "MSH.21",
								"valueIdentifier": {
									"extension": [
										{
											"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/assigning-authority",
											"extension": [
												{
													"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id",
													"valueString": "2.16.840.1.113883.9.82"
												},
												{
													"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id-type",
													"valueCode": "ISO"
												}
											]
										}
									],
									"value": "LAB_PRU_COMPONENT"
								}
							},
							{
								"url": "MSH.21",
								"valueIdentifier": {
									"extension": [
										{
											"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/assigning-authority",
											"extension": [
												{
													"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id",
													"valueString": "2.16.840.1.113883.9.22"
												},
												{
													"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id-type",
													"valueCode": "ISO"
												}
											]
										}
									],
									"value": "LAB_TO_COMPONENT"
								}
							}
						]
					}
				],
				"eventCoding": {
					"system": "http://terminology.hl7.org/CodeSystem/v2-0003",
					"code": "O01",
					"display": "ORM^O01^ORM_O01"
				},
				"destination": [
					{
						"extension": [
							{
								"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id",
								"valueString": "natus.health.state.la.us"
							},
							{
								"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id-type",
								"valueString": "DNS"
							},
							{
								"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2Field",
								"valueString": "MSH.5"
							}
						],
						"name": "NATUS",
						"receiver": {
							"reference": "Organization/1729025485467113959.e064cfb3-9574-40b8-8104-3d1ce4efcf36"
						}
					}
				],
				"sender": {
					"reference": "Organization/1729025485465452364.5658f2d4-cad8-467e-ba72-420c399de29f"
				},
				"source": {
					"extension": [
						{
							"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/namespace-id",
							"valueString": "Epic"
						},
						{
							"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id",
							"valueString": "1.2.840.114350.1.13.145.2.7.2.695071"
						},
						{
							"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id-type",
							"valueString": "ISO"
						},
						{
							"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2Field",
							"valueString": "MSH.3"
						}
					],
					"endpoint": "urn:oid:1.2.840.114350.1.13.145.2.7.2.695071"
				}
			}
		},
		{
			"fullUrl": "Organization/1729025485465452364.5658f2d4-cad8-467e-ba72-420c399de29f",
			"resource": {
				"resourceType": "Organization",
				"id": "1729025485465452364.5658f2d4-cad8-467e-ba72-420c399de29f",
				"identifier": [
					{
						"extension": [
							{
								"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2Field",
								"valueString": "HD.1"
							}
						],
						"value": "Ochsner Health"
					},
					{
						"extension": [
							{
								"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2Field",
								"valueString": "HD.2,HD.3"
							}
						],
						"type": {
							"coding": [
								{
									"system": "http://terminology.hl7.org/CodeSystem/v2-0301",
									"code": "DNS"
								}
							]
						},
						"value": "ochsner.org"
					}
				]
			}
		},
		{
			"fullUrl": "Organization/1729025485467113959.e064cfb3-9574-40b8-8104-3d1ce4efcf36",
			"resource": {
				"resourceType": "Organization",
				"id": "1729025485467113959.e064cfb3-9574-40b8-8104-3d1ce4efcf36",
				"extension": [
					{
						"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2Field",
						"valueString": "MSH.6"
					}
				],
				"identifier": [
					{
						"extension": [
							{
								"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2Field",
								"valueString": "HD.1"
							}
						],
						"value": "LA Public Health Lab"
					},
					{
						"extension": [
							{
								"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2Field",
								"valueString": "HD.2,HD.3"
							}
						],
						"type": {
							"coding": [
								{
									"system": "http://terminology.hl7.org/CodeSystem/v2-0301",
									"code": "ISO"
								}
							]
						},
						"system": "urn:ietf:rfc:3986",
						"value": "2.16.840.1.114222.4.3.26.3.2"
					}
				]
			}
		},
		{
			"fullUrl": "Patient/1729025485514475854.ea6692df-6be8-4a1a-a2a2-8bfc2b0a6d31",
			"resource": {
				"resourceType": "Patient",
				"id": "1729025485514475854.ea6692df-6be8-4a1a-a2a2-8bfc2b0a6d31",
				"extension": [
					{
						"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/pid-patient",
						"extension": [
							{
								"url": "PID.8"
							},
							{
								"url": "PID.24",
								"valueString": "N"
							},
							{
								"url": "PID.30",
								"valueString": "N"
							}
						]
					},
					{
						"url": "http://hl7.org/fhir/StructureDefinition/patient-mothersMaidenName",
						"valueHumanName": {
							"extension": [
								{
									"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/xpn-human-name",
									"extension": [
										{
											"url": "XPN.2",
											"valueString": "SADIE"
										},
										{
											"url": "XPN.3",
											"valueString": "S"
										}
									]
								}
							],
							"family": "SMITH",
							"given": [
								"SADIE",
								"S"
							]
						}
					}
				],
				"identifier": [
					{
						"extension": [
							{
								"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/cx-identifier",
								"extension": [
									{
										"url": "CX.5",
										"valueString": "MR"
									}
								]
							},
							{
								"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2Field",
								"valueString": "PID.3"
							}
						],
						"type": {
							"coding": [
								{
									"code": "MR"
								}
							]
						},
						"value": "11102779",
						"assigner": {
							"reference": "Organization/1729025485472875178.1c30131d-828f-485d-81d8-c1f54a629c91"
						}
					}
				],
				"name": [
					{
						"extension": [
							{
								"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/xpn-human-name",
								"extension": [
									{
										"url": "XPN.2",
										"valueString": "BB SARAH"
									},
									{
										"url": "XPN.7",
										"valueString": "L"
									}
								]
							}
						],
						"use": "official",
						"family": "SMITH",
						"given": [
							"BB SARAH"
						]
					}
				],
				"telecom": [
					{
						"extension": [
							{
								"url": "http://hl7.org/fhir/StructureDefinition/contactpoint-area",
								"valueString": "763"
							},
							{
								"url": "http://hl7.org/fhir/StructureDefinition/contactpoint-local",
								"valueString": "5555555"
							},
							{
								"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/xtn-contact-point",
								"extension": [
									{
										"url": "XTN.3",
										"valueString": "PH"
									},
									{
										"url": "XTN.7",
										"valueString": "5555555"
									},
									{
										"url": "XTN.9",
										"valueString": "(763)555-5555"
									}
								]
							}
						],
						"system": "phone",
						"use": "home"
					}
				],
				"gender": "male",
				"birthDate": "2024-06-30",
				"_birthDate": {
					"extension": [
						{
							"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2-date-time",
							"valueString": "20240630131023"
						},
						{
							"url": "http://hl7.org/fhir/StructureDefinition/patient-birthTime",
							"valueDateTime": "2024-06-30T13:10:23Z"
						}
					]
				},
				"deceasedBoolean": false,
				"address": [
					{
						"extension": [
							{
								"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/xad-address",
								"extension": [
									{
										"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/sad-address-line",
										"extension": [
											{
												"url": "SAD.1",
												"valueString": "555 STATE HIGHWAY 13"
											}
										]
									},
									{
										"url": "XAD.7",
										"valueCode": "H"
									}
								]
							}
						],
						"use": "home",
						"line": [
							"555 STATE HIGHWAY 13"
						],
						"city": "DEER CREEK",
						"district": "OTTER TAIL",
						"state": "LA",
						"postalCode": "56527-9657",
						"country": "USA"
					}
				],
				"multipleBirthInteger": 1
			}
		},
		{
			"fullUrl": "Organization/1729025485472875178.1c30131d-828f-485d-81d8-c1f54a629c91",
			"resource": {
				"resourceType": "Organization",
				"id": "1729025485472875178.1c30131d-828f-485d-81d8-c1f54a629c91",
				"identifier": [
					{
						"extension": [
							{
								"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2Field",
								"valueString": "HD.1"
							}
						],
						"value": "CRPMRN"
					}
				]
			}
		},
		{
			"fullUrl": "ServiceRequest/1729025485561374087.7b19bb67-dde7-4903-a5e7-b30cf081141a",
			"resource": {
				"resourceType": "ServiceRequest",
				"id": "1729025485561374087.7b19bb67-dde7-4903-a5e7-b30cf081141a",
				"extension": [
					{
						"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/business-event",
						"valueCode": "NW"
					},
					{
						"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/business-event",
						"valueString": "20240702052913"
					},
					{
						"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/orc-common-order",
						"extension": [
							{
								"url": "orc-21-ordering-facility-name",
								"valueReference": {
									"reference": "Organization/1729025485548641587.fc7ca7d6-5683-4067-bb5a-bef81cadfbeb"
								}
							},
							{
								"url": "orc-21-ordering-facility-name",
								"valueReference": {
									"reference": "Organization/1729025485552000012.4e24a435-b26e-4dab-82ce-e9591b23c00a"
								}
							},
							{
								"url": "orc-12-ordering-provider",
								"valueReference": {
									"reference": "Practitioner/1729025485555340408.4003adf4-3b0e-4ad7-b359-7b1f81041938"
								}
							}
						]
					},
					{
						"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/obr-observation-request",
						"extension": [
							{
								"url": "OBR.2",
								"valueIdentifier": {
									"extension": [
										{
											"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/assigning-authority",
											"extension": [
												{
													"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/namespace-id",
													"valueString": "EPIC"
												},
												{
													"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id",
													"valueString": "1.2.840.114350.1.13.145.2.7.2.695071"
												},
												{
													"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id-type",
													"valueCode": "ISO"
												}
											]
										}
									],
									"value": "421832901"
								}
							}
						]
					}
				],
				"identifier": [
					{
						"extension": [
							{
								"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2Field",
								"valueString": "ORC.2"
							},
							{
								"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/assigning-authority",
								"extension": [
									{
										"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/namespace-id",
										"valueString": "EPIC"
									},
									{
										"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id",
										"valueString": "1.2.840.114350.1.13.145.2.7.2.695071"
									},
									{
										"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id-type",
										"valueCode": "ISO"
									}
								]
							}
						],
						"type": {
							"coding": [
								{
									"system": "http://terminology.hl7.org/CodeSystem/v2-0203",
									"code": "PLAC"
								}
							]
						},
						"value": "421832901"
					}
				],
				"status": "unknown",
				"subject": {
					"reference": "Patient/1729025485514475854.ea6692df-6be8-4a1a-a2a2-8bfc2b0a6d31"
				},
				"authoredOn": "2024-07-02T05:29:13Z",
				"_authoredOn": {
					"extension": [
						{
							"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2-date-time",
							"valueString": "20240702052913"
						}
					]
				},
				"requester": {
					"reference": "PractitionerRole/1729025485516648010.ec6b9123-c963-4b89-aa9c-7fa2031fb019"
				}
			}
		},
		{
			"fullUrl": "Organization/1729025485517809731.0f8fccfb-4d6d-4f92-9710-abd6eb4a2a52",
			"resource": {
				"resourceType": "Organization",
				"id": "1729025485517809731.0f8fccfb-4d6d-4f92-9710-abd6eb4a2a52",
				"identifier": [
					{
						"extension": [
							{
								"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2Field",
								"valueString": "HD.1"
							}
						],
						"value": "NPI"
					}
				]
			}
		},
		{
			"fullUrl": "Practitioner/1729025485530608174.a00c7197-be6a-4a22-b82b-54355430f9da",
			"resource": {
				"resourceType": "Practitioner",
				"id": "1729025485530608174.a00c7197-be6a-4a22-b82b-54355430f9da",
				"extension": [
					{
						"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/assigning-authority",
						"extension": [
							{
								"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/namespace-id",
								"valueString": "NPI"
							},
							{
								"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id-unknown-type"
							},
							{
								"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id-type"
							}
						]
					},
					{
						"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/xcn-practitioner",
						"extension": [
							{
								"url": "XCN.3",
								"valueString": "JANE"
							},
							{
								"url": "XCN.10",
								"valueString": "L"
							}
						]
					},
					{
						"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2Field",
						"valueString": "ORC.12"
					}
				],
				"identifier": [
					{
						"type": {
							"coding": [
								{
									"extension": [
										{
											"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/codeable-concept-id",
											"valueBoolean": true
										}
									],
									"code": "NPI"
								}
							]
						},
						"value": "1265136360",
						"assigner": {
							"reference": "Organization/1729025485517809731.0f8fccfb-4d6d-4f92-9710-abd6eb4a2a52"
						}
					}
				],
				"name": [
					{
						"use": "official",
						"family": "JONES",
						"given": [
							"JANE"
						]
					}
				]
			}
		},
		{
			"fullUrl": "Organization/1729025485533303279.42c9728f-2120-480e-a403-c08e70a19d47",
			"resource": {
				"resourceType": "Organization",
				"id": "1729025485533303279.42c9728f-2120-480e-a403-c08e70a19d47",
				"extension": [
					{
						"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/organization-name-type",
						"valueCoding": {
							"extension": [
								{
									"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/cwe-coding",
									"valueCodeableConcept": {
										"extension": [
											{
												"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2Field",
												"valueString": "XON.2"
											}
										]
									}
								}
							]
						}
					},
					{
						"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/xon-organization",
						"extension": [
							{
								"url": "XON.10",
								"valueString": "1043269798"
							}
						]
					}
				],
				"identifier": [
					{
						"extension": [
							{
								"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/assigning-authority",
								"extension": [
									{
										"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/namespace-id",
										"valueString": "CMS"
									},
									{
										"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id-unknown-type"
									},
									{
										"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id-type"
									}
								]
							}
						],
						"type": {
							"coding": [
								{
									"system": "http://terminology.hl7.org/CodeSystem/v2-0203",
									"code": "NPI"
								}
							]
						},
						"value": "1043269798"
					}
				],
				"name": "ST. CLOUD HOSPITAL"
			}
		},
		{
			"fullUrl": "PractitionerRole/1729025485516648010.ec6b9123-c963-4b89-aa9c-7fa2031fb019",
			"resource": {
				"resourceType": "PractitionerRole",
				"id": "1729025485516648010.ec6b9123-c963-4b89-aa9c-7fa2031fb019",
				"practitioner": {
					"reference": "Practitioner/1729025485530608174.a00c7197-be6a-4a22-b82b-54355430f9da"
				},
				"organization": {
					"reference": "Organization/1729025485533303279.42c9728f-2120-480e-a403-c08e70a19d47"
				}
			}
		},
		{
			"fullUrl": "Organization/1729025485548641587.fc7ca7d6-5683-4067-bb5a-bef81cadfbeb",
			"resource": {
				"resourceType": "Organization",
				"id": "1729025485548641587.fc7ca7d6-5683-4067-bb5a-bef81cadfbeb",
				"extension": [
					{
						"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/organization-name-type",
						"valueCoding": {
							"extension": [
								{
									"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/cwe-coding",
									"valueCodeableConcept": {
										"extension": [
											{
												"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2Field",
												"valueString": "XON.2"
											}
										]
									}
								}
							]
						}
					},
					{
						"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/xon-organization",
						"extension": [
							{
								"url": "XON.10",
								"valueString": "1043269798"
							}
						]
					}
				],
				"identifier": [
					{
						"extension": [
							{
								"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/assigning-authority",
								"extension": [
									{
										"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/namespace-id",
										"valueString": "CMS"
									},
									{
										"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id-unknown-type"
									},
									{
										"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id-type"
									}
								]
							}
						],
						"type": {
							"coding": [
								{
									"system": "http://terminology.hl7.org/CodeSystem/v2-0203",
									"code": "NPI"
								}
							]
						},
						"value": "1043269798"
					}
				],
				"name": "ST. CLOUD HOSPITAL"
			}
		},
		{
			"fullUrl": "Organization/1729025485552000012.4e24a435-b26e-4dab-82ce-e9591b23c00a",
			"resource": {
				"resourceType": "Organization",
				"id": "1729025485552000012.4e24a435-b26e-4dab-82ce-e9591b23c00a",
				"extension": [
					{
						"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/organization-name-type",
						"valueCoding": {
							"extension": [
								{
									"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/cwe-coding",
									"valueCodeableConcept": {
										"extension": [
											{
												"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2Field",
												"valueString": "XON.2"
											}
										]
									}
								}
							]
						}
					},
					{
						"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/xon-organization",
						"extension": [
							{
								"url": "XON.10",
								"valueString": "362"
							}
						]
					}
				],
				"identifier": [
					{
						"extension": [
							{
								"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/assigning-authority",
								"extension": [
									{
										"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/namespace-id",
										"valueString": "LA Public Health Lab"
									},
									{
										"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id-unknown-type"
									},
									{
										"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id-type"
									}
								]
							}
						],
						"type": {
							"coding": [
								{
									"system": "http://terminology.hl7.org/CodeSystem/v2-0203",
									"code": "Submitter ID"
								}
							]
						},
						"value": "362"
					}
				],
				"name": "ST. CLOUD HOSPITAL"
			}
		},
		{
			"fullUrl": "Organization/1729025485553189702.c5316349-9ec1-431d-9b43-53ef67fab64f",
			"resource": {
				"resourceType": "Organization",
				"id": "1729025485553189702.c5316349-9ec1-431d-9b43-53ef67fab64f",
				"identifier": [
					{
						"extension": [
							{
								"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2Field",
								"valueString": "HD.1"
							}
						],
						"value": "NPI"
					}
				]
			}
		},
		{
			"fullUrl": "Practitioner/1729025485555340408.4003adf4-3b0e-4ad7-b359-7b1f81041938",
			"resource": {
				"resourceType": "Practitioner",
				"id": "1729025485555340408.4003adf4-3b0e-4ad7-b359-7b1f81041938",
				"extension": [
					{
						"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/assigning-authority",
						"extension": [
							{
								"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/namespace-id",
								"valueString": "NPI"
							},
							{
								"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id-unknown-type"
							},
							{
								"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id-type"
							}
						]
					},
					{
						"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/xcn-practitioner",
						"extension": [
							{
								"url": "XCN.3",
								"valueString": "JANE"
							},
							{
								"url": "XCN.10",
								"valueString": "L"
							}
						]
					}
				],
				"identifier": [
					{
						"type": {
							"coding": [
								{
									"extension": [
										{
											"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/codeable-concept-id",
											"valueBoolean": true
										}
									],
									"code": "NPI"
								}
							]
						},
						"value": "1265136360",
						"assigner": {
							"reference": "Organization/1729025485553189702.c5316349-9ec1-431d-9b43-53ef67fab64f"
						}
					}
				],
				"name": [
					{
						"use": "official",
						"family": "JONES",
						"given": [
							"JANE"
						]
					}
				]
			}
		},
		{
			"fullUrl": "Observation/1729025485575710084.efc88881-ad56-4396-b3e2-59a0a7b95b46",
			"resource": {
				"resourceType": "Observation",
				"id": "1729025485575710084.efc88881-ad56-4396-b3e2-59a0a7b95b46",
				"extension": [
					{
						"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/sub-id",
						"valueString": "1"
					},
					{
						"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/obx-observation",
						"extension": [
							{
								"url": "OBX.2",
								"valueId": "NM"
							},
							{
								"url": "OBX.6"
							},
							{
								"url": "OBX.11",
								"valueString": "O"
							}
						]
					}
				],
				"status": "unknown",
				"subject": {
					"reference": "Patient/1729025485514475854.ea6692df-6be8-4a1a-a2a2-8bfc2b0a6d31"
				},
				"effectiveDateTime": "2024-07-02T05:00:00Z",
				"_effectiveDateTime": {
					"extension": [
						{
							"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2-date-time",
							"valueString": "20240702050000"
						}
					]
				},
				"valueQuantity": {
					"value": 1769.859285,
					"unit": "gram",
					"system": "UCUM",
					"code": "g"
				}
			}
		},
		{
			"fullUrl": "Observation/1729025485586870867.36913699-172a-443f-bf3e-f65021361e4e",
			"resource": {
				"resourceType": "Observation",
				"id": "1729025485586870867.36913699-172a-443f-bf3e-f65021361e4e",
				"extension": [
					{
						"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/sub-id",
						"valueString": "1"
					},
					{
						"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/obx-observation",
						"extension": [
							{
								"url": "OBX.2",
								"valueId": "NM"
							},
							{
								"url": "OBX.6"
							},
							{
								"url": "OBX.11",
								"valueString": "O"
							}
						]
					}
				],
				"status": "unknown",
				"subject": {
					"reference": "Patient/1729025485514475854.ea6692df-6be8-4a1a-a2a2-8bfc2b0a6d31"
				},
				"effectiveDateTime": "2024-07-02T05:00:00Z",
				"_effectiveDateTime": {
					"extension": [
						{
							"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2-date-time",
							"valueString": "20240702050000"
						}
					]
				},
				"valueQuantity": {
					"value": 32,
					"unit": "week",
					"system": "UCUM",
					"code": "wk"
				}
			}
		},
		{
			"fullUrl": "Observation/1729025485599436854.3d7eaf7f-da3a-46e4-b7ad-984669d259b5",
			"resource": {
				"resourceType": "Observation",
				"id": "1729025485599436854.3d7eaf7f-da3a-46e4-b7ad-984669d259b5",
				"extension": [
					{
						"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/sub-id",
						"valueString": "1"
					},
					{
						"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/obx-observation",
						"extension": [
							{
								"url": "OBX.2",
								"valueId": "CWE"
							},
							{
								"url": "OBX.11",
								"valueString": "O"
							}
						]
					}
				],
				"status": "unknown",
				"subject": {
					"reference": "Patient/1729025485514475854.ea6692df-6be8-4a1a-a2a2-8bfc2b0a6d31"
				},
				"effectiveDateTime": "2024-07-02T05:00:00Z",
				"_effectiveDateTime": {
					"extension": [
						{
							"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2-date-time",
							"valueString": "20240702050000"
						}
					]
				},
				"valueCodeableConcept": {
					"coding": [
						{
							"extension": [
								{
									"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/cwe-coding",
									"valueString": "coding"
								},
								{
									"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/cwe-coding-system",
									"valueString": "LN"
								}
							],
							"system": "http://loinc.org",
							"code": "LA12419-0",
							"display": "INFANT IN NICU AT TIME OF SPECIMEN COLLECTION"
						}
					]
				}
			}
		},
		{
			"fullUrl": "Observation/1729025485616641829.f76244ae-7c11-457b-863b-73e757fcb179",
			"resource": {
				"resourceType": "Observation",
				"id": "1729025485616641829.f76244ae-7c11-457b-863b-73e757fcb179",
				"extension": [
					{
						"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/sub-id",
						"valueString": "1"
					},
					{
						"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/obx-observation",
						"extension": [
							{
								"url": "OBX.2",
								"valueId": "CWE"
							},
							{
								"url": "OBX.11",
								"valueString": "O"
							}
						]
					}
				],
				"status": "unknown",
				"subject": {
					"reference": "Patient/1729025485514475854.ea6692df-6be8-4a1a-a2a2-8bfc2b0a6d31"
				},
				"effectiveDateTime": "2024-07-02T05:00:00Z",
				"_effectiveDateTime": {
					"extension": [
						{
							"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2-date-time",
							"valueString": "20240702050000"
						}
					]
				},
				"valueCodeableConcept": {
					"coding": [
						{
							"extension": [
								{
									"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/cwe-coding",
									"valueString": "coding"
								},
								{
									"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/cwe-coding-system",
									"valueString": "99LDH"
								}
							],
							"code": "N",
							"display": "No"
						}
					]
				}
			}
		},
		{
			"fullUrl": "Observation/1729025485631768068.49fc3473-4de0-4918-9cc1-dd44e7ebb869",
			"resource": {
				"resourceType": "Observation",
				"id": "1729025485631768068.49fc3473-4de0-4918-9cc1-dd44e7ebb869",
				"extension": [
					{
						"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/sub-id",
						"valueString": "1"
					},
					{
						"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/obx-observation",
						"extension": [
							{
								"url": "OBX.2",
								"valueId": "CWE"
							},
							{
								"url": "OBX.11",
								"valueString": "O"
							}
						]
					}
				],
				"status": "unknown",
				"subject": {
					"reference": "Patient/1729025485514475854.ea6692df-6be8-4a1a-a2a2-8bfc2b0a6d31"
				},
				"effectiveDateTime": "2024-07-02T05:00:00Z",
				"_effectiveDateTime": {
					"extension": [
						{
							"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2-date-time",
							"valueString": "20240702050000"
						}
					]
				},
				"valueCodeableConcept": {
					"coding": [
						{
							"extension": [
								{
									"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/cwe-coding",
									"valueString": "coding"
								},
								{
									"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/cwe-coding-system",
									"valueString": "99LDH"
								}
							],
							"code": "N",
							"display": "No"
						}
					]
				}
			}
		},
		{
			"fullUrl": "Observation/1729025485650656489.e9fe9911-660a-41b7-b79f-e894ef6719a0",
			"resource": {
				"resourceType": "Observation",
				"id": "1729025485650656489.e9fe9911-660a-41b7-b79f-e894ef6719a0",
				"extension": [
					{
						"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/sub-id",
						"valueString": "1"
					},
					{
						"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/obx-observation",
						"extension": [
							{
								"url": "OBX.2",
								"valueId": "CWE"
							},
							{
								"url": "OBX.11",
								"valueString": "O"
							}
						]
					}
				],
				"status": "unknown",
				"subject": {
					"reference": "Patient/1729025485514475854.ea6692df-6be8-4a1a-a2a2-8bfc2b0a6d31"
				},
				"effectiveDateTime": "2024-07-02T05:00:00Z",
				"_effectiveDateTime": {
					"extension": [
						{
							"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2-date-time",
							"valueString": "20240702050000"
						}
					]
				},
				"valueCodeableConcept": {
					"coding": [
						{
							"extension": [
								{
									"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/cwe-coding",
									"valueString": "coding"
								},
								{
									"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/cwe-coding-system",
									"valueString": "99LDH"
								}
							],
							"code": "N",
							"display": "No"
						}
					]
				}
			}
		},
		{
			"fullUrl": "Observation/1729025485659204292.ccd42d6a-0dab-435f-ad51-f141f6efa32f",
			"resource": {
				"resourceType": "Observation",
				"id": "1729025485659204292.ccd42d6a-0dab-435f-ad51-f141f6efa32f",
				"extension": [
					{
						"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/sub-id",
						"valueString": "1"
					},
					{
						"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/obx-observation",
						"extension": [
							{
								"url": "OBX.2",
								"valueId": "CWE"
							},
							{
								"url": "OBX.11",
								"valueString": "O"
							}
						]
					}
				],
				"status": "unknown",
				"subject": {
					"reference": "Patient/1729025485514475854.ea6692df-6be8-4a1a-a2a2-8bfc2b0a6d31"
				},
				"effectiveDateTime": "2024-07-02T05:00:00Z",
				"_effectiveDateTime": {
					"extension": [
						{
							"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2-date-time",
							"valueString": "20240702050000"
						}
					]
				},
				"valueCodeableConcept": {
					"coding": [
						{
							"extension": [
								{
									"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/cwe-coding",
									"valueString": "coding"
								},
								{
									"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/cwe-coding-system",
									"valueString": "LN"
								}
							],
							"system": "http://loinc.org",
							"code": "LA12418-2",
							"display": "TPN"
						}
					]
				}
			}
		},
		{
			"fullUrl": "Observation/1729025485675501133.09c64d22-13dc-406d-929c-21a687a55bac",
			"resource": {
				"resourceType": "Observation",
				"id": "1729025485675501133.09c64d22-13dc-406d-929c-21a687a55bac",
				"extension": [
					{
						"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/sub-id",
						"valueString": "2"
					},
					{
						"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/obx-observation",
						"extension": [
							{
								"url": "OBX.2",
								"valueId": "CWE"
							},
							{
								"url": "OBX.11",
								"valueString": "O"
							}
						]
					}
				],
				"status": "unknown",
				"subject": {
					"reference": "Patient/1729025485514475854.ea6692df-6be8-4a1a-a2a2-8bfc2b0a6d31"
				},
				"effectiveDateTime": "2024-07-02T05:00:00Z",
				"_effectiveDateTime": {
					"extension": [
						{
							"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2-date-time",
							"valueString": "20240702050000"
						}
					]
				},
				"valueCodeableConcept": {
					"coding": [
						{
							"extension": [
								{
									"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/cwe-coding",
									"valueString": "coding"
								},
								{
									"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/cwe-coding-system",
									"valueString": "LN"
								}
							],
							"system": "http://loinc.org",
							"code": "LA16914-6",
							"display": "BREAST MILK"
						}
					]
				}
			}
		},
		{
			"fullUrl": "Observation/1729025485692589010.a234f132-3c37-48ce-8f8f-f9903b96a2fd",
			"resource": {
				"resourceType": "Observation",
				"id": "1729025485692589010.a234f132-3c37-48ce-8f8f-f9903b96a2fd",
				"extension": [
					{
						"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/sub-id",
						"valueString": "1"
					},
					{
						"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/obx-observation",
						"extension": [
							{
								"url": "OBX.2",
								"valueId": "CWE"
							},
							{
								"url": "OBX.11",
								"valueString": "O"
							}
						]
					}
				],
				"status": "unknown",
				"subject": {
					"reference": "Patient/1729025485514475854.ea6692df-6be8-4a1a-a2a2-8bfc2b0a6d31"
				},
				"effectiveDateTime": "2024-07-02T05:00:00Z",
				"_effectiveDateTime": {
					"extension": [
						{
							"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2-date-time",
							"valueString": "20240702050000"
						}
					]
				},
				"valueCodeableConcept": {
					"coding": [
						{
							"extension": [
								{
									"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/cwe-coding",
									"valueString": "coding"
								}
							],
							"code": "0516199364"
						}
					]
				}
			}
		},
		{
			"fullUrl": "Observation/1729025485698738603.e458ba2f-2c05-4c41-9040-ec304ce523cb",
			"resource": {
				"resourceType": "Observation",
				"id": "1729025485698738603.e458ba2f-2c05-4c41-9040-ec304ce523cb",
				"extension": [
					{
						"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/sub-id",
						"valueString": "1"
					},
					{
						"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/obx-observation",
						"extension": [
							{
								"url": "OBX.2",
								"valueId": "ST"
							},
							{
								"url": "OBX.11",
								"valueString": "O"
							}
						]
					}
				],
				"status": "unknown",
				"subject": {
					"reference": "Patient/1729025485514475854.ea6692df-6be8-4a1a-a2a2-8bfc2b0a6d31"
				},
				"effectiveDateTime": "2024-07-02T05:00:00Z",
				"_effectiveDateTime": {
					"extension": [
						{
							"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2-date-time",
							"valueString": "20240702050000"
						}
					]
				},
				"valueString": "Daniel Davis/218-555-1000"
			}
		},
		{
			"fullUrl": "Observation/1729025485715583997.c526c8b5-5929-4f9e-8fd7-5f87922bf7e8",
			"resource": {
				"resourceType": "Observation",
				"id": "1729025485715583997.c526c8b5-5929-4f9e-8fd7-5f87922bf7e8",
				"extension": [
					{
						"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/sub-id",
						"valueString": "1"
					},
					{
						"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/obx-observation",
						"extension": [
							{
								"url": "OBX.2",
								"valueId": "ST"
							},
							{
								"url": "OBX.11",
								"valueString": "O"
							}
						]
					}
				],
				"status": "unknown",
				"subject": {
					"reference": "Patient/1729025485514475854.ea6692df-6be8-4a1a-a2a2-8bfc2b0a6d31"
				},
				"effectiveDateTime": "2024-07-02T05:00:00Z",
				"_effectiveDateTime": {
					"extension": [
						{
							"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2-date-time",
							"valueString": "20240702050000"
						}
					]
				},
				"valueString": "DAVIS, DANIEL, MD"
			}
		},
		{
			"fullUrl": "Observation/1729025485742914496.54508934-8e62-4d7c-a313-adf80135fd92",
			"resource": {
				"resourceType": "Observation",
				"id": "1729025485742914496.54508934-8e62-4d7c-a313-adf80135fd92",
				"extension": [
					{
						"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/sub-id",
						"valueString": "1"
					},
					{
						"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/obx-observation",
						"extension": [
							{
								"url": "OBX.2",
								"valueId": "ST"
							},
							{
								"url": "OBX.11",
								"valueString": "O"
							}
						]
					}
				],
				"status": "unknown",
				"subject": {
					"reference": "Patient/1729025485514475854.ea6692df-6be8-4a1a-a2a2-8bfc2b0a6d31"
				},
				"effectiveDateTime": "2024-07-02T05:00:00Z",
				"_effectiveDateTime": {
					"extension": [
						{
							"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2-date-time",
							"valueString": "20240702050000"
						}
					]
				},
				"valueString": "Lakeridge Health Revere"
			}
		},
		{
			"fullUrl": "Specimen/1729025485779948986.52d1645b-1ad3-4f4b-885f-bf3fe59c5b9f",
			"resource": {
				"resourceType": "Specimen",
				"id": "1729025485779948986.52d1645b-1ad3-4f4b-885f-bf3fe59c5b9f",
				"extension": [
					{
						"url": "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2Segment",
						"valueString": "SPM"
					}
				]
			}
		}
	]
}
```

**Comment by jbiskie** [comment]
ReportStream HL7 to FHIR output of https://github.com/CDCgov/trusted-intermediary/tree/main/examples/MN/003_MN_ORM_O01_NBS.hl7 when run on the OML-Updates-Base feature branch:

[LA ORM converted to FHIR on OML-Updates-Branch.json](https://github.com/user-attachments/files/17385026/LA.ORM.converted.to.FHIR.on.OML-Updates-Branch.json)

**Comment by basiliskus** [comment]
I submitted `003_MN_ORM_O01_NBS.hl7` locally using the `OML-Updates-Base` branch in RS which has the new ORM/OML mappings. It looks like this branch fixed most of the issues we were seeing. For `003_MN_ORM_O01_NBS.hl7`, the only difference is that one of the OBRs is removed in the final HL7 and OBR-7 and OBR-15 are missing for the one preserved. Here's the content:
```
MSH|^~\&|Epic^1.2.840.114350.1.13.145.2.7.2.695071^ISO|Centracare^centracare.com^DNS|NATUS^natus.health.state.mn.us^DNS|MN Public Health Lab^2.16.840.1.114222.4.1.10080^ISO|20230506052916-0500||OML^O21^OML_O21|31808297|D|2.5.1|||AL|AL|||||LAB_PRU_COMPONENT^^2.16.840.1.113883.9.82^ISO~LAB_TO_COMPONENT^^2.16.840.1.113883.9.22^ISO
PID|1||11102779^^^CRPMRN^MR||SMITH^BB SARAH^^^^^L|SMITH^SADIE^S|20230504131023-0500|M|||555 STATE HIGHWAY 13^^DEER CREEK^MN^56527-9657^USA^H^^OTTER TAIL||^^PH^^^763^5555555^^(763)555-5555|||||||||||N|1|||||N
ORC|NW|421832901^EPIC^1.2.840.114350.1.13.145.2.7.2.695071^ISO|||||||20230506052913-0500|||1265136360^JONES^JANE^^^^^^NPI^L^^^NPI|||||||||ST. CLOUD HOSPITAL^L^^^^CMS^NPI^^^1043269798~ST. CLOUD HOSPITAL^L^^^^MN Public Health Lab^Submitter ID^^^739
OBR|1|421832901^EPIC^1.2.840.114350.1.13.145.2.7.2.695071^ISO||54089-8^Newborn screening panel American Health Information Community (AHIC)^LN|||||||P
OBX|1|NM|8339-4^BIRTH WEIGHT MEASURED^LN|1|1769.859285|g^gram^UCUM|||||O|||20230506050000-0500|||||||||||||||QST|AOE
OBX|2|NM|57714-8^OBSTETRIC ESTIMATION OF GESTATIONAL AGE^LN|1|32|wk^week^UCUM|||||O|||20230506050000-0500|||||||||||||||QST|AOE
OBX|3|CWE|57713-0^INFANT FACTORS THAT AFFECT NEWBORN SCREENING INTERPRETATION^LN|1|LA12419-0^INFANT IN NICU AT TIME OF SPECIMEN COLLECTION^LN||||||O|||20230506050000-0500|||||||||||||||QST|AOE
OBX|4|CWE|MNDSIB^DOES THE BABY HAVE A DECEASED SIBLING?^99MDH|1|N^No^99MDH||||||O|||20230506050000-0500|||||||||||||||QST|AOE
OBX|5|CWE|MNDEFECT^DOES THE BABY HAVE BIRTH DEFECTS?^99MDH|1|N^No^99MDH||||||O|||20230506050000-0500|||||||||||||||QST|AOE
OBX|6|CWE|MNFAM^FAMILY HISTORY OF DISORDER ON MN SCREENING PANEL^99MDH|1|N^No^99MDH||||||O|||20230506050000-0500|||||||||||||||QST|AOE
OBX|7|CWE|67704-7^FEEDING TYPES^LN|1|LA12418-2^TPN^LN||||||O|||20230506050000-0500|||||||||||||||QST|AOE
OBX|8|CWE|67704-7^FEEDING TYPES^LN|2|LA16914-6^BREAST MILK^LN||||||O|||20230506050000-0500|||||||||||||||QST|AOE
OBX|9|CWE|57723-9^UNIQUE BAR CODE NUMBER OF CURRENT SAMPLE^LN|1|0516199364||||||O|||20230506050000-0500|||||||||||||||QST|AOE
OBX|10|ST|62328-0^DISCHARGE PROVIDER PRACTICE PHONE NUMBER^LN|1|Daniel Davis/218-555-1000||||||O|||20230506050000-0500|||||||||||||||QST|AOE
OBX|11|ST|62324-9^POST-DISCHARGE PROVIDER NAME^LN|1|DAVIS, DANIEL, MD||||||O|||20230506050000-0500|||||||||||||||QST|AOE
OBX|12|ST|62326-4^POST-DISCHARGE PROVIDER PRACTICE NAME^LN|1|Lakeridge Health Revere||||||O|||20230506050000-0500|||||||||||||||QST|AOE
```

**Comment by GilmoreA6** [comment]
The 2nd OBR is being dropped because 003_MN_ORM_O01_NBS.hl7 isn't a valid ORM per HL7 V2.5.1.
2.5.1 specifies that the ORC segment is required for each order group, adding a second ORC above the second OBR produces both OBRs:

```
MSH|^~\&|Epic^1.2.840.114350.1.13.145.2.7.2.695071^ISO|Centracare^centracare.com^DNS|NATUS^natus.health.state.mn.us^DNS|MN Public Health Lab^2.16.840.1.114222.4.1.10080^ISO|20230506052916-0500||ORM^O01^ORM_O01|31808297|D|2.5.1|||AL|AL|||||LAB_PRU_COMPONENT^^2.16.840.1.113883.9.82^ISO~LAB_TO_COMPONENT^^2.16.840.1.113883.9.22^ISO
PID|1||11102779^^^CRPMRN^MR||SMITH^BB SARAH^^^^^L|SMITH^SADIE^S|20230504131023-0500|M|||555 STATE HIGHWAY 13^^DEER CREEK^MN^56527-9657^USA^H^^OTTER TAIL||^^PH^^^763^5555555^^(763)555-5555|||||||||||N|1|||||N
ORC|NW|421832901^EPIC^1.2.840.114350.1.13.145.2.7.2.695071^ISO|||||||20230506052913-0500|||1265136360^JONES^JANE^^^^^^NPI^L^^^NPI|||||||||ST. CLOUD HOSPITAL^L^^^^CMS^NPI^^^1043269798~ST. CLOUD HOSPITAL^L^^^^MN Public Health Lab^Submitter ID^^^739
OBR|1|421832901^EPIC^1.2.840.114350.1.13.145.2.7.2.695071^ISO||54089-8^Newborn screening panel American Health Information Community (AHIC)^LN|||||||P
ORC|NW|421832901^EPIC^1.2.840.114350.1.13.145.2.7.2.695071^ISO|||||||20230506052913-0500|||1265136360^JONES^JANE^^^^^^NPI^L^^^NPI|||||||||ST. CLOUD HOSPITAL^L^^^^CMS^NPI^^^1043269798~ST. CLOUD HOSPITAL^L^^^^MN Public Health Lab^Submitter ID^^^739
OBR|2|421832901^EPIC^1.2.840.114350.1.13.145.2.7.2.695071^ISO||57717-1^Newborn screen card data panel^LN
OBX|1|NM|8339-4^BIRTH WEIGHT MEASURED^LN|1|1769.859285|g^gram^UCUM|||||O|||20230506050000-0500|||||||||||||||QST|AOE
OBX|2|NM|57714-8^OBSTETRIC ESTIMATION OF GESTATIONAL AGE^LN|1|32|wk^week^UCUM|||||O|||20230506050000-0500|||||||||||||||QST|AOE
OBX|3|CWE|57713-0^INFANT FACTORS THAT AFFECT NEWBORN SCREENING INTERPRETATION^LN|1|LA12419-0^INFANT IN NICU AT TIME OF SPECIMEN COLLECTION^LN||||||O|||20230506050000-0500|||||||||||||||QST|AOE
OBX|4|CWE|MNDSIB^DOES THE BABY HAVE A DECEASED SIBLING?^99MDH|1|N^No^99MDH||||||O|||20230506050000-0500|||||||||||||||QST|AOE
OBX|5|CWE|MNDEFECT^DOES THE BABY HAVE BIRTH DEFECTS?^99MDH|1|N^No^99MDH||||||O|||20230506050000-0500|||||||||||||||QST|AOE
OBX|6|CWE|MNFAM^FAMILY HISTORY OF DISORDER ON MN SCREENING PANEL^99MDH|1|N^No^99MDH||||||O|||20230506050000-0500|||||||||||||||QST|AOE
OBX|7|CWE|67704-7^FEEDING TYPES^LN|1|LA12418-2^TPN^LN||||||O|||20230506050000-0500|||||||||||||||QST|AOE
OBX|8|CWE|67704-7^FEEDING TYPES^LN|2|LA16914-6^BREAST MILK^LN||||||O|||20230506050000-0500|||||||||||||||QST|AOE
OBX|9|CWE|57723-9^UNIQUE BAR CODE NUMBER OF CURRENT SAMPLE^LN|1|0516199364||||||O|||20230506050000-0500|||||||||||||||QST|AOE
OBX|10|ST|62328-0^DISCHARGE PROVIDER PRACTICE PHONE NUMBER^LN|1|Daniel Davis/218-555-1000||||||O|||20230506050000-0500|||||||||||||||QST|AOE
OBX|11|ST|62324-9^POST-DISCHARGE PROVIDER NAME^LN|1|DAVIS, DANIEL, MD||||||O|||20230506050000-0500|||||||||||||||QST|AOE
OBX|12|ST|62326-4^POST-DISCHARGE PROVIDER PRACTICE NAME^LN|1|Lakeridge Health Revere||||||O|||20230506050000-0500|||||||||||||||QST|AOE
```

**Comment by basiliskus** [comment]
Submitting the LA example `examples/LA/001_LA_ORM_O01_NBS.hl7`, the final HL7 is:
- Missing both `NK1` segments
- Missing `OBR-7` and `OBR-15`
- Added `P` value in `OBR-11`

**Comment by GilmoreA6** [comment]
additional information regarding above:

- the NK1 segments are missing in the output because NK1 is not a valid segment in an ORM-001 message but is a valid segment in an OML-021. The segments are getting lost because RS has no mapping for NK1 to the "RelatedPerson" FHIR resource for an ORM message. If the NK1 segment needs to be passed for LA, we would need to create separate mappings for this or have that information put into a valid segment for an ORM message.
- The "P" is explicitly being added in receiver transform "etor-oml-receiver-transform.yml". This can be easily changed.
- OBR-7 is not being populated because the FHIR-> HL7 mappings for an OML_021 expects this tp be populated with information from the FHIR "Specimen" resource which is not created because ORM messages do not contain an SPM segment
- OBR-15 is not being populated because 2.5.1 OML-021 messages only have the OBR-15 field for backwards compatibility so there is not mapping for this field.

**Comment by brick-green** [comment]
@GilmoreA6 Is going to make a new ticket for the NK1 work listed above. 

What is the plan for receiver transform "etor-oml-receiver-transform.yml" changes? Is that needed to close out this ticket?

**Comment by basiliskus** [comment]
@brick-green  good point. We need to figure out if we need that transformation for OBR-11

**Comment by basiliskus** [comment]
@brick-green Confirmed that we should remove that transformation, so that task should be part of this ticket

**Comment by brick-green** [comment]
@basiliskus @GilmoreA6  Can you add that task above? Can you also review the other tasks and check off the ones that are complete?

**Comment by basiliskus** [comment]
@brick-green added it

**Comment by GilmoreA6** [comment]
@brick-green there is a task  "Look into possibility of adding assertions in the RS Integration Test framework to catch mapping issues" to me that also seems like a separate issue, since this ticket is specifically about validating the RS mappings and reporting gaps, thoughts?

**Comment by brick-green** [comment]
The integration testing will be broken out into a new ticket(s). Remaining work is removing the transformation file.

**Comment by basiliskus** [comment]
Added this [PR in RS](https://github.com/CDCgov/prime-reportstream/pull/16444) to remove the `etor-oml-receiver-transform.yml` transform

**Comment by JohnNKing** [comment]
Discussed in Stream 2 Refinement in relation to #472; we're assuming at this point that SPM will be provided, which would provide an alternate means of getting the info from OBR-15.

**Comment by basiliskus** [comment]
PR to remove `etor-oml-receiver-transform.yml` in RS has been merged

**Comment by JohnNKing** [comment]
Moving back to Review / QA until the ReportStream work has been merged (to main) and deployed to staging.

**Comment by JohnNKing** [comment]
@JohnNKing If not yet captured, we need stories created for:
1. Inbound SPM mapping
2. OBR-7 and OBR-15 support

**Comment by brick-green** [comment]
@jbiskie Did you or someone else try changing the MSH.9 to OML then seeing what happens?

**Comment by jbiskie** [comment]
> @jbiskie Did you or someone else try changing the MSH.9 to OML then seeing what happens?

@GilmoreA6 Am I recalling correctly that you talked about doing this?

**Comment by GilmoreA6** [comment]
Putting the new LA order ORM messages through ReportStream changing MSH-9 from "ORM_001^ORM_001" to "OML^O21^OML_O21" results in an OML message that is complete except for the following missing segments/fields:

Missing Segments:
PD1 
IN1
GT1 

Missing Fields
PID-2
PID-18 (Intentional, this is SSN and is removed on purpose)
ORC-7
OBR-27
SPM-35 

The missing fields are mostly (if not entirely) due to them not existing in the HL7 v2 2.5.1 standards

**Comment by brick-green** [comment]
Ochsner made the change to MSH.9 and emailed a new message!! Can we get a quick test in with this one? @jbiskie @GilmoreA6 

```
MSH|^~\&|13^1.2.840.114350.1.13.286.2.7.2.695071^ISO|4169|Natus.LA.STAG^2.16.840.1.114222.4.3.26.3.2|Louisiana Office of Public Health Laboratory^2.16.840.1.114222.4.3.22.17.1.1|20241115111935|739|OML^O21|31|T|2.5.1|||AL|NE

PID|1|10453874^^^MRN^MRN|10453874^^^MRN^MRN||PKUSS^B BOY BEAKER||20241108110000|M||2054-5^Black or African-American^CDCREC|1202 COOLIDGE ST.^^LAFAYETTE^LA^70503^US^P^^LAFAYETTE|LAFAYETTE|(557)559-5559^P^PH||ENGLISH|Single|||000-00-0000||PKUSS^BEAKER^MOM^|HISPANIC||Y|2|||||N

PD1||||1184608010^DREWITZ^TROY^U.^^^^^NPI^^^^NPI

NK1|1|PKUSS^BEAKER^MOM^|Mother|1202 Coolidge St.^^LAFAYETTE^LA^70503^US|(557)559-5559^^PH

PV1|1||^8288^24827^^^^^^^^||||||||||||||1184608010^DREWITZ^TROY^U.^^^^^NPI^^^^NPI

IN1|1|114415|308114401|BLUE CROSS BLUE SHIELD|P O BOX 98029^^BATON ROUGE^LA^70898-9029^||(800)495-2583|||||||||PKUSS^B^BOY BEAKER^|Self|20241108|1202 COOLIDGE ST.^^LAFAYETTE^LA^70503^US^^^LAFAYETTE|||1||||||||||||||||||||None|M

GT1|1|100251816|PKUSS^BEAKER^MOM^||1202 COOLIDGE ST.^^LAFAYETTE^LA^70503^US^^^LAFAYETTE|(557)559-5559^P^PH||19891224|F|P/F|MOT|999-99-9999||||||||Full

ORC|NW|243217750^EPC||71414751|||^^^20241113125400^^Routine^^||20241113125406|739^BEAKER^PHLEBOTOMIST^^||1184608010^DREWITZ^TROY^U.^^^^^NPI^^^^NPI|^^^^^^^^OLGH LABOR AND DELIVERY|(504)391-8896||||OCH155315^OCH155315|||OCHSNER LAFAYETTE GENERAL MEDICAL HOSPITAL^D^^^^OCHSNER HEALTH SYSTEM^XX^^^30100|1214 COOLIDGE BLVD^^LAFAYETTE^LA^70503-2621^US^B^^LAFAYETTE|(337)289-7991^^PH|120 OCHSNER BLVD^SUITE 360^GRETNA^LA^70056^^C|||||I|VERBAL WITH

OBR|1|243217750^EPC|1000319696^Beaker|54089-8^Newborn screening panel American Health Information Community (AHIC)^LN^^^^2.72||20241113125400|20241113131841|||739^BEAKER^PHLEBOTOMIST^^|Lab Collect|||||1184608010^DREWITZ^TROY^U.^^^^^NPI^^^^NPI|(504)391-8896|||193988||||Lab|||^^^20241113125400^^Routine^^|||||||||20241113125400

OBX|1|ST|57714-8^Gestational Age (Weeks + Days)||382||||||F|||20241113125402|||||||||||||||QST

OBX|2|ST|82312-0^Infant Blood Transfusion?||Yes||||||F|||20241113125402|||||||||||||||QST

OBX|3|DT|62317-3^Date of Transfusion||20241108||||||F|||20241113125402|||||||||||||||QST

OBX|4|ST|67704-7^All Feeding Since Birth||Human Milk \T\ Formula||||||F|||20241113125402|||||||||||||||QST

OBX|5|ST|PLT3280^Name of Formula(s)||Enfomil||||||F|||20241113125402|||||||||||||||QST

SPM|1|^24LADH-318X0007&Beaker||BLOOD^Blood||||BLOOD^Blood|||||||||20241113131841|20241113131842||Y|||||||||||||||S
```

**Comment by GilmoreA6** [comment]
@brick-green it looks like ReportStream throws an error if MSH-9-3 is not populated.

{"error": "Internal error at 2024-11-15T17:46:23.069054272Z"}

**Comment by brick-green** [comment]
Got it. I'll let them know we also need "OML_021" in MSH-9.3

**Comment by basiliskus** [comment]
I know RS uses MSH-9.3 to identify the message type, so that's more important to have than MSH-9.1/2

**Comment by GilmoreA6** [comment]
Reason for Blocked Tagged: This card depends on PR https://github.com/CDCgov/prime-reportstream/pull/16539/files which is not yet merged

**Comment by basiliskus** [comment]
One last PR should be open soon to merge changes into main. The work is being tracked in this story: https://github.com/CDCgov/prime-reportstream/issues/16065

**Comment by GilmoreA6** [comment]
draft PR for merging into main has been started https://github.com/CDCgov/prime-reportstream/pull/16659

**Comment by GilmoreA6** [comment]
After the OML branch has been merged to main, here are the current missing segments/fields

Missing Segments:
IN1
GT1

Missing Fields
PID-2 - Missing because PID-2 is "Backwards Compatible" field in 2.5.1
PID.19 and PID.20 - are deprecated in the HL7v2.7 and NIST HL7v2.5.1 specs. Further, the HAPI v2.7 model has set both # fields to NULLDT. 
PD1.4 - Missing because PD1-1 is "Backwards Compatible" field in 2.5.1
ORC-7 - Missing because ORC-7 is "Backwards Compatible" field in 2.5.1
OBR-27- Missing because OBR-27 is "Backwards Compatible" field in 2.5.1
SPM-35 - Missing because SPM-35 does not exist in 2.5.1

**Comment by brick-green** [comment]
So IN1, GT1, and PD1 is ticketed. 

Created [new ticket](https://github.com/orgs/CDCgov/projects/45/views/2?pane=issue&itemId=88921184&issue=CDCgov%7Ctrusted-intermediary%7C1626) for the remaining missing fields listed above.

Any remaining tasks for this ticket?

**Comment by GilmoreA6** [comment]
@brick-green I don't think so, I will move to done
